sdk/identity/identity-cache-persistence/CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
+
+### New features
+
+- This release marks the initial beta availability of the `@azure/identity-cache-persistence` package. This package provides an extension to `@azure/identity` that enables persistent token caching in a secure storage defined by the Operating System (caching of `access_token` values across sessions), which allows skipping interactive authentication flows if a previously cached `access_token` or `refresh_token` are still available. This is implemented using the following technologies:
```

Comment:
I don’t think there’s need to mention details about `access_token` or `refresh_token` here. Can we make it a bit more abstract? Something like:

> enables persistent caching in a secure storage defined by the Operative System, which allows skipping interactive authentication flows after an initial authentication has succeeded

(please refine based on your perspective)

Reply:
I would like to mention "tokens", but don't have any strong feelings about "access token" or "refresh token" specifically. I made some changes to this. Let me know what you think.